cypress/e2e/news/createReadDeleteNews.feature

```diff
@@ -0,0 +1,80 @@
+@release
+@stable_test
+Feature:  News - To read a news on the respective dashboards
+
+  As a teacher I want to read the news shown on the dashboard so that I'm informed about the latest news
+
+  @stable_test
```

Comment:
Why mentioning the tag `@stable_test` again here when its already used in higher level?